mailer.http.class.php

```diff
@@ -219,9 +219,14 @@ protected function encode_attachment($data)
         return base64_encode($data);
     }
 
-    protected function read_attachment($path)
+    protected function read_attachment($data)
     {
-        return file_get_contents($path);
+        if ( file_exists( $data ) ) {
```

Comment:
Thanks for the PR. Do you think [`is_file`](http://php.net/manual/en/function.is-file.php) is more appropriate, in terms of how it sounds at least. Also please add a comment of explaining why we're doing it so next person understands it quickly.

Reply:
Should be good now :+1:

Good call on `is_file()`. Didn't realize `file_exists()` would return true on directories.

Reply:
Thank you very much, I'll test it in next week and merge it.

Reply:
Sorry, I couldn't manage time to test it yet. Hoping to do it today/tomorrow.